### PR TITLE
Enable Raspberry Touch 2 rotation with overlay

### PR DESCRIPTION
**Pull request by foonerd**
The panel-ilitek-ili9881c.c which Touch Display 2 is based on already contains orientation.
By minor update to the vc4-kms-dsi-ili9881-7inch-overlay.dts panel can be rotated with minimal effort.

Issue opened here: 
https://github.com/raspberrypi/documentation/issues/3923

Complete discussion and working dbto in the community thread:
https://community.volumio.com/t/guide-official-raspberry-touch-display-2/69791

### CODE REVIEW

**Comment by 6by9** [comment]
Commits should include a commit text and `Signed-off-by:` line., but otherwise looks reasonable.

**Comment by foonerd** [comment]
@6by9 
For my benefit, the commit line should be:
dtoverlays: Enable Raspberry Touch 2 rotation

Did I get this right?

Who should be listed as "Signed-off-by: "?

File: arch/arm/boot/dts/overlays/README [hunk 1]
```diff
@@ -5249,6 +5249,7 @@ Params: sizex                   Touchscreen size x (default 720)
         invy                    Touchscreen inverted y axis
         swapxy                  Touchscreen swapped x y axis
         disable_touch           Disables the touch screen overlay driver
+        rotatation              Display rotation {0,90,180,270} (default 0)
```
Comment (6by9) [inline arch/arm/boot/dts/overlays/README]: As the dtoverlaycheck docs noticed, typo in `rotatation`

**Review by 6by9 — COMMENTED** [review]
`Signed-off-by:` is documented in https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin

It should match the commit author in this case. For upstream both would need use your real name rather than any nickname, but for dtoverlays that isn't as critical.

File: arch/arm/boot/dts/overlays/README (hunk 1 above)
Comment (foonerd) [reply arch/arm/boot/dts/overlays/README]: Corrected.

**Comment by 6by9** [comment]
The patches get applied to the tree exactly as they are presented in the PR. The 3 changes therefore want to get squashed into one, and force pushed (`git push -f ... `) to your Github branch.

**Comment by foonerd** [comment]
I see. Lets close this PR and open another, correct one. - Do you agree?

**Comment by pelwell** [comment]
No - if the end result is one patch then we can squash at the point of merging.

**Comment by foonerd** [comment]
Thanks. In such case, is there anything else needed from me at this point?

**Comment by pelwell** [comment]
I think we're good - I'll just give it the once over then merge.

**Comment by harmjanr** [comment]
Any clue when we can expect this PR / 6.6.63 via `sudo apt full-upgrade`?

**Comment by 6by9** [comment]
> Any clue when we can expect this PR / 6.6.63 via `sudo apt full-upgrade`?

Unlikely to be before Christmas as we've just done a kernel release.

**Pull request merged**